### DISCUSSION TITLE
mettre le lien de la discussion dans le readme

### DISCUSSION THREAD

**Discussion by devcrafting** [cnumr/best-practices#75] [General]
Pour chaque BP, vu que l'idée est d'avoir une discussion, on pourrait mettre le lien de la discussion de laBP (ça marche aussi si on venait à avoir plusieurs discussions sur une BP 😉)

Dans le même ordre d'idée, on pourra voir le lien de la/des PR d'une BP.

**Comment by ACTLEM** [comment]
Tu es dur :-D ... Pour l'instant, nous mettons celles en discussion, si on a du temps on pourra ajouter le lien mais la recherche par numéro fonctionne bien aussi ;-)
Sinon toutes les BP ne sont pas forcément à discuter, si l'actuel convient (je pense qu'il y en a) il n'y a pas besoin.

Par contre, il y aura bien un lien vers le ou les issues (et donc les indirectement les PR associées) sur la page d'accueil.

**Reply by devcrafting** [reply]
Je pensais juste à mettre le lien quand la discussion est créée et que quelqu'un change le picto ;)
C'est top l'organisation du dépôt, aucun reproche 👌
NB: en effet je fais la recherche de l'id pour être sûr qu'une discussion n'existe pas déjà
